### PR TITLE
added hints to dev console commands

### PR DESCRIPTION
**Pull request by ludoza**

### CODE REVIEW

File: src/MenuDevConsole.cpp [hunk 1]
```diff
@@ -233,6 +234,7 @@ void MenuDevConsole::execute() {
 		}
 		else {
 			log_history->add(msg->get("ERROR: Too few arguments"), false, &color_error);
+			log_history->add(msg->get("HINT: " + args[0] + " <category> [<x> <y>]"), false, &color_hint);
```
Comment (dorkster) [inline src/MenuDevConsole.cpp]: `x` and `y` should be removed here, as `spawn_enemy` only supports specifying enemy category.

**Comment by dorkster** [comment]
I like this, but please refer to the [note](https://github.com/clintbellanger/flare-engine/pull/1384/files#r28205796) I added.

**Comment by ludoza** [comment]
Saw it to late(my bad), so decided to add it. Not sure if we should remove min and max level?

**Comment by dorkster** [comment]
It's okay to remove min/max level for now, since that functionality is broken when using `spawn_enemy` from the dev console. Since it goes through, PowerManager's `spawn` function the level range is discarded. I'll have to do some refactoring there.

File: src/MenuDevConsole.cpp (hunk 1 above)
Comment (ludoza) [inline src/MenuDevConsole.cpp]: added functionality to support x and y

**Comment by dorkster** [comment]
Thanks, it just needed one more [fix](https://github.com/clintbellanger/flare-engine/commit/2ddf982c78dde29e0ce268fcbea168ec35e33ab2).

**Pull request merged**